yb-voyager/cmd/export.go

```diff
@@ -392,18 +392,36 @@ func saveExportTypeInMSR() {
 }
 
 func checkDependenciesForExport() (binaryCheckIssues []string, err error) {
-	if source.DBType == POSTGRESQL {
+	var missingTools []string
+	switch source.DBType {
+	case POSTGRESQL:
 		sourceDBVersion := source.DB().GetVersion()
-		for _, binary := range pgExportDependencies {
+		for _, binary := range pgExportCommands {
 			_, binaryCheckIssue, err := srcdb.GetAbsPathOfPGCommandAboveVersion(binary, sourceDBVersion)
 			if err != nil {
 				return nil, err
 			} else if binaryCheckIssue != "" {
 				binaryCheckIssues = append(binaryCheckIssues, binaryCheckIssue)
 			}
 		}
+
+		missingTools = utils.CheckTools("strings")
+
+	case MYSQL:
```

Comment:
minor improvement: for mysql, oracle, you can skip the ora2pg check if it's live migration. 
(add a todo for now, because as I see this function is called in lots of places - exportschema, exportdata, assess-migration)